pkg/pillar/pubsub/doc.go

```diff
@@ -50,13 +50,13 @@
 //
 // The driver is responsible for implementing the underlying mechanics of
 // publishing and subscribing. While `pubsub.PubSub` and its components -
-// `Publication` and `Subscription` - handle the in-memory and diff aspects,
+// `NewPublication` and `NewSubscription` - handle the in-memory and diff aspects,
```

Comment:
No, this is wrong. `Publication` and `Subscription` handle the in-memory... 

`NewPublication` and `NewSubscription` are just the function calls that _create_ the `Publication` and `Subscription`, which handle the in-memory and...